### PR TITLE
Update French strings

### PR DESCRIPTION
**Pull request by Ilithy**
New attempt to upgrade the French translation :)

_Really sorry_

### CODE REVIEW

**Comment by Ilithy** [comment]
I don't understand why the build fails, I checked the file, I don't find any problem...did I delete one line too many (in the untranslatable strings maybe?)

**Comment by jooooscha** [comment]
No problem. Probably not even you fault. I will look at it later locally

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -37,15 +37,25 @@
...
     <string name="light">Clair</string>
     <string name="double_tap_lock_uninstall_message">Tapoter deux fois pour verrouiller est activé
         \n\nRemarque: Désactivez-le d\'abord si vous décidez de supprimer Olauncher.</string>
     <string name="accessibility_service_description">L\'autorisation d\'accès au service d\'accessibilité est nécessaire pour activer le double tapotement pour verrouiller. Aucunes données acquises par le service ne quitteront pas votre appareil.</string>
-    <string name="lock_layout_description">verrouiller la description de la disposition en utilisant un identifiant unique pour verrouiller l\'écran</string>
+    <string name="lock_layout_description">verrouiller la disposition en utilisant un identifiant unique pour verrouiller l\'écran</string>
     <string name="open">Ouvrir le lien</string>
     <string name="copy">Copier</string>
     <string name="close">Fermer</string>
+    <string name="app_language">Langue de l'\interface</string>
```
Comment (jooooscha) [inline app/src/main/res/values-fr/strings.xml:51]: Found it :). It should be `\'` not `'\`

Comment (Ilithy) [reply app/src/main/res/values-fr/strings.xml]: Thank you very much, I have just corrected the error, it was indeed due to a mistake on my part, my apologies

Comment (jooooscha) [reply app/src/main/res/values-fr/strings.xml]: No problem. In Germany we say, "Passiert den Besten": happens to the best ;)

Comment (Ilithy) [reply app/src/main/res/values-fr/strings.xml]: 💘

**Pull request merged**